common/config/seq.go

```diff
@@ -0,0 +1,62 @@
+package config
+
+import (
+	"github.com/odigos-io/odigos/common"
+)
+
+const (
+	SEQ_ENDPOINT = "SEQ_ENDPOINT"
+	SEQ_API_KEY  = "SEQ_API_KEY"
+)
+
+type Seq struct{}
+
+func (j *Seq) DestType() common.DestinationType {
+	return common.SeqDestinationType
+}
+
+func (j *Seq) ModifyConfig(dest ExporterConfigurer, currentConfig *Config) ([]string, error) {
+	config := dest.GetConfig()
+	uniqueUri := "seq-" + dest.GetID()
+	var pipelineNames []string
+
+	endpoint, exists := config[SEQ_ENDPOINT]
+	if !exists {
+		return nil, errorMissingKey(SEQ_ENDPOINT)
+	}
+	endpoint, err := parseOtlpHttpEndpoint(endpoint)
+	if err != nil {
+		return nil, err
+	}
+	if !urlHostContainsPort(endpoint) {
+		endpoint += ":5341/ingest/otlp"
```

Comment:
should we assert that the path is empty before appending to it? users might get confused and paste the entire http endpoint here in which case there will be no error and the collector will fail to start.

Consider extending `parseOtlpHttpEndpoint` to also verify path, or make another assertion for that